triton/plan.sh

```diff
@@ -0,0 +1,38 @@
+pkg_name=triton
+pkg_origin=core
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MPL-2.0')
+pkg_deps=(core/node)
+pkg_build_deps=(core/jq-static core/sed)
+pkg_bin_dirs=(bin)
+pkg_description="Joyent Triton CLI and client"
+pkg_upstream_url="https://www.joyent.com/triton"
+
+pkg_version () {
+  npm view triton --json | jq --raw-output .version
```

Comment:
![tenor-170041422](https://cloud.githubusercontent.com/assets/1130349/26754111/c6b63b46-4839-11e7-914d-de95262a23ac.gif)
I think this means we could periodically just build this plan and get the latest version without having to manually check for a new version.